email_login/src/components/UpgradeToPlusPopup.js

```diff
@@ -13,7 +13,7 @@ const UpgradeToPlusPopup = props => {
         <h1>{upgradePlus.title}</h1>
       </div>
       <div className="popup-paragraph">
-        <p>{string.formatString(upgradePlus.message, props.maxDevices)}</p>
+        <p>{string.formatString(upgradePlus.message, 2)}</p>
```

Comment:
probably is good to maintain in props.maxDevices (ideally if is STANDARD the number would be 2), because if we change that in the future

Reply:
2 is not the maximum devices, it is my current max limit, which the server does not give us. Anyways I decided to remove it entirely, and handle the message as is